x-pack/test/search_sessions_integration/tests/apps/management/search_sessions/sessions_management.ts

```diff
@@ -113,68 +113,79 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
         const searchSessionList = await PageObjects.searchSessionsManagement.getList();
         expect(searchSessionList.length).to.be(10);
-        expectSnapshot(searchSessionList.map((ss) => [ss.app, ss.name, ss.created, ss.expires]))
-          .toMatchInline(`
+        expectSnapshot(
```

Comment:
Expected changes because these changes change now displayed in an expired state. (Also covers a function change of this pr)